flash_cards/views.py

```diff
@@ -1,7 +1,23 @@
 from django.shortcuts import render
-from django.contrib.auth.decorators import login_required, user_passes_test 
+from django.contrib.auth.decorators import login_required, user_passes_test
+
+from flash_cards.models import FlashCard, Deck
```

Comment:
```suggestion
from django.contrib.auth.decorators import login_required, user_passes_test
from flash_cards.models import FlashCard, Deck
```
consider cutting this line for the sake of brevity